source/sdk/os/Time.ooc

```diff
@@ -135,7 +135,7 @@ Time: class {
         }
         return -1
     }
-	
+
```

Comment:
Additions are fine, just avoid modifications. As for moving `DateTimeData` into `Time`, I think you should be able to `extend` `Time` instead. Move back `DateTimeData` into the same file where `DateTime` is located, and add something like
```ooc
extend Time {
    dateTimeData: static func -> DateTimeData {
...
```
This is done in several places with e.g. `String`, but I'm not sure it's going to work with static functions. Worth a try though.